numcodecs/tests/test_checksum32.py

```diff
@@ -17,6 +14,12 @@
     check_repr,
 )
 
+has_crc32c = False
+with suppress(ImportError):
```

Comment:
Instead of siliently skipping cr32c tests, I think it's clearer if they are explicitly skipped by pytest (so it's visible and easier to debug when they are skipped). I think the easiest way to do this would be to parameterize the tests that you've had to modify, and then have a conditional `pytest.skip` call inside the tests if the parmeterized parameter is related to cr32c, and the package isn't available.